### PR TITLE
Added TC in convert test file, invalid inference

### PR DESCRIPTION
**Pull request by amitsrivastava78**
Added TC for the invalid interence type

### CODE REVIEW

**Comment by amitsrivastava78** [comment]
@gargn  can you pls review the PR and let me know your comments

File: tensorflow/lite/python/convert_test.py [hunk 1]
```diff
@@ -68,6 +68,20 @@ def testQuantization(self):
         quantized_input_stats=[(0., 1.)])
     self.assertTrue(tflite_model)
 
+  def testInferenceTypeInvalid(self):
```
Comment (gargn) [inline tensorflow/lite/python/convert_test.py]: nit: Please move this test after `testQuantizationInvalid`

**Review by gargn — CHANGES_REQUESTED** [review]

File: tensorflow/lite/python/convert_test.py (hunk 1 above)
Comment (amitsrivastava78) [reply tensorflow/lite/python/convert_test.py]: @gargn thanks alot for spending the time on the review, i have updated as per your comments, kindly approve the same.

**Comment by amitsrivastava78** [comment]
@gargn & @rthadur , I have updated the code as per your comments, kindly check, if everything is ok, kindly approve the PR.

**Comment by amitsrivastava78** [comment]
@rthadur , can you pls update the label to review requested

**Review by gargn — APPROVED** [review]

**Comment by amitsrivastava78** [comment]
@gargn , thanks for approving the PR, can you please help to get this merged.

Regards
Amit

**Comment by amitsrivastava78** [comment]
@rthadur , can you please help to get this merged.

Regards
Amit

**Comment by amitsrivastava78** [comment]
@rthadur , can you please help to get this merged.

Regards
Amit

**Comment by gargn** [comment]
It seems support for `int8` was added so this test no longer causes a failure.

**Pull request closed without merging**